### PR TITLE
add Ariadne Visual Media Service

### PR DESCRIPTION
**Pull request by emanueldima**
For 3d object visualisation, with the official site and valid certificate.

### CODE REVIEW

File: tools/AriadneVisualMediaService.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+    "task": "3D Visualisation",
+    "deployment": "production",
+    "name": "Ariadne Visual Media Service",
+    "logo": "YourLogoComesHere.png",
```
Comment (andmor-) [inline tools/AriadneVisualMediaService.json:5]: Missing logo file

File: tools/AriadneVisualMediaService.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+    "task": "3D Visualisation",
+    "deployment": "production",
```
Comment (andmor-) [inline tools/AriadneVisualMediaService.json:3]: Maybe "development" for now?

File: schemas/spec-v1.schema.json [hunk 3]
```diff
@@ -31,6 +31,7 @@
       "type": "string",
       "title": "Task description",
       "enum": [
+        "3D Visualisation",
```
Comment (andmor-) [inline schemas/spec-v1.schema.json:34]: Not sure. Is this something for the Language Resource Switchboard? I would like to hear from others as well

File: tools/AriadneVisualMediaService.json (hunk 2 above)
Comment (emanueldima) [reply tools/AriadneVisualMediaService.json]: fixed

File: schemas/spec-v1.schema.json (hunk 3 above)
Comment (emanueldima) [reply schemas/spec-v1.schema.json:34]: This is meant for data repositories with 3d scans of historical artefacts. Not related to language but social sciences.

**Pull request merged**